### PR TITLE
Added multiple digit seven segment display

### PR DESCRIPTION
**Pull request by suppergerrie2**
Added a seven segment display which supports multiple digits: https://editor.p5js.org/full/SkqBEHGs7

### CODE REVIEW

**Comment by suppergerrie2** [comment]
Yeah but I can't see what is wrong, some new line?

File: _CodingChallenges/117-seven-segment.md [hunk 1]
```diff
@@ -15,6 +15,14 @@ videos:
   - title: "What's The Longest Word You Can Write With Seven-Segment Displays?"
     author: "Tom Scott"
     video_id: "zp4BMR88260"
+
```
Comment (nilsweb) [inline _CodingChallenges/117-seven-segment.md]: This line just contains whitespace. That's why it's failing! 😜

**Comment by nilsweb** [comment]
Nice work! 👍🏻

**Comment by suppergerrie2** [comment]
I like how github adds them automatically lol

**Comment by nilsweb** [comment]
Speaking of things that GitHub does automatically: It would be awesome if the tests could mark the lines that contain errors in the PR by itself! I'll have to look into that... 🤔

**Pull request merged**